files/image_config/platform/rc.local

```diff
@@ -11,6 +11,9 @@
 #
 # By default this script does nothing.
 
+eval SONIC_VERSION=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
```

Comment:
>SONIC_VERSION [](start = 5, length = 13)

The cat/grep way will include quotation mark.
Try sonic-cfggen -y /etc/sonic/sonic_version.yml -v build_version.

Reply:
I didn't write this, I just moved it. The use of `eval` evaluates the expression inside the quotation marks. I can change and test it if you'd like.